os/fs/driver/mtd/smart.c

```diff
@@ -18,7 +18,7 @@
 /****************************************************************************
  * fs/driver/mtd/smart.c
  *
- * Sector Mapped Allocation for Really Tiny (SMART) Flash block driver.
+ * Journaling Sector Mapped Allocation for Really Tiny (jSmartFs) Flash block driver.
```

Comment:
The smart.c driver initially written for SmartFs driver interfacing now handles our jSmartFs driver interfacing.
Hence, changing this part is tricky.
@Taejun-Kwon I'd like your suggestions too please.

There would not be any need for changing references at this level but journaling mechanism is implemented at this layer itself.

My suggestion is that this line should not be changed and rather a new line explaining the transition from smartfs to jSmartFs in brief be added.